src/libponyc/codegen/genprim.c

```diff
@@ -485,7 +485,7 @@ void genprim_pointer_methods(compile_t* c, reach_type_t* t)
   pointer_lt(c, t);
 }
```

Comment:
i dont think we want to do these changes as part of RFC. 

need to discuss at sync.